backend/plugin/mfa/otp/otp.go

```diff
@@ -0,0 +1,103 @@
+// Package otp is the plugin for OTP Multi-Factor Authentication.
+package otp
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pquerna/otp/totp"
+	"golang.org/x/exp/slices"
+)
+
+const (
+	// issuerName is the name of the issuer of the OTP token.
+	issuerName = "Bytebase"
+	// secondsInMinute is the number of seconds in a minute.
+	secondsInMinute = 60
+	// maxPastSecretCount is the maximum number of past secret we will check.
+	maxPastSecretCount = 5
+)
+
+// removeSecondsFromTimestamp removes the seconds from the timestamp.
+func removeSecondsFromTimestamp(timestamp time.Time) int64 {
+	truncated := time.Date(timestamp.Year(), timestamp.Month(), timestamp.Day(), timestamp.Hour(), timestamp.Minute(), 0, 0, timestamp.Location())
```

Comment:
Can we convert to unix time (need to check it's unit) divided by 60 or 60000?

Reply:
Updated. PTAL